test/e2e/providers/azure/azure.go

```diff
@@ -33,19 +32,27 @@ type Provider struct {
 	oc *clusterinfo.OpenShift
 	*config.InfrastructureStatus
 	vmSize string
+	// vmLocation is the Azure "location" in which VMs should be created. This should not be read directly, instead
+	// it should be accessed through getVMLocation().
+	vmLocation string
 }
 
 // New returns a new Azure provider struct with the give client set and ssh key pair
-func New(clientset *clusterinfo.OpenShift, infraStatus *config.InfrastructureStatus) *Provider {
+func New(clientset *clusterinfo.OpenShift, infraStatus *config.InfrastructureStatus) (*Provider, error) {
```

Comment:
Is there a case where the constructor can return error? IMO no need for this.

Reply:
You're right, was left over from a previous method I was trying